uint16 gpio_id
uint16 frequency
---
bool has_succeeded